pallets/file-system/src/utils.rs

```diff
@@ -1652,5 +1895,25 @@ mod hooks {
 
             remaining_weight.saturating_reduce(potential_weight);
         }
+
+        fn process_expired_move_bucket_request(
+            msp_id: ProviderIdFor<T>,
+            bucket_id: BucketIdFor<T>,
+            remaining_weight: &mut Weight,
+        ) {
+            let db_weight = T::DbWeight::get();
+            let potential_weight = db_weight.reads_writes(0, 1);
```

Comment:
It's not just one write though, it's two.

Reply:
Thanks for spotting.